tools/parity_scale_decoder/src/generate_map.rs

```diff
@@ -469,9 +469,9 @@ mod tests {
     fn schemas_types_is_a_subset_of_map_types() {
         // These types **shouldn't** implement `Decode`. As such we need to make an exception.
         let exceptions = HashSet::from(type_names_arr![
-            Vec<iroha_core::genesis::GenesisTransaction>,
-            iroha_core::genesis::GenesisTransaction,
-            iroha_core::genesis::RawGenesisBlock,
+            Vec<iroha_genesis::GenesisTransaction>,
+            iroha_genesis::GenesisTransaction,
+            iroha_genesis::RawGenesisBlock,
```

Comment:
is genesis required as a dependency of kagami?

Reply:
yes, we need it in tests